components/nav.js

```diff
@@ -52,124 +54,109 @@ const ListItem = glamorous.li({
   paddingBottom: 4,
 })
 
-// Use withTheme with glamorous.Ul, or this ?
-const List = glamorous.ul((props, {colors, mediaQueries}) => ({
-  listStyle: 'none',
-  display: 'block',
-  justifyContent: 'center',
-  alignItems: 'center',
-  fontSize: '1.25em',
-  margin: '0 auto',
+const List = glamorous.ul(
+  // eslint-disable-next-line complexity
+  ({top, isOpen, theme: {colors, mediaQueries}}) => ({
+    listStyle: 'none',
+    display: 'block',
+    fontSize: '1.25em',
+    margin: '0 auto',
+    paddingLeft: top ? null : 0,
 
-  height: 'auto',
-  width: '100%',
-  padding: props.isOpen ? '1rem 0' : 0,
-  maxHeight: props.isOpen ? '100%' : 0,
-  opacity: props.isOpen ? 1 : 0,
-  overflow: 'hidden',
-  textAlign: 'center',
-  backgroundColor: colors.white,
-  [mediaQueries.mediumUp]: {
-    display: 'flex',
-    justifyContent: 'center',
-    width: 'auto',
-    maxHeight: '4rem',
-    backgroundColor: 'inherit',
-    opacity: 1,
-  },
-}))
+    height: 'auto',
+    overflow: 'hidden',
+    backgroundColor: colors.white,
+    [mediaQueries.mediumDown]: {
+      textAlign: 'center',
+      width: '100%',
+      padding: isOpen ? '1rem 0' : 0,
+      maxHeight: isOpen ? '100%' : 0,
+      opacity: isOpen ? 1 : 0,
+    },
+    [mediaQueries.mediumUp]: {
+      display: 'flex',
+      justifyContent: top ? 'center' : 'flex-start',
+      flexDirection: top ? 'row' : 'column',
```

Comment:
With this line the mobile version is breaking on any page we don't update to have `top={true}`. Since this is a new prop only index and Getting Started are correct on mobile right now.